orbit/cmd/desktop/desktop.go

```diff
@@ -110,6 +111,19 @@ func main() {
 		log.Info().Msgf("got a TUF update root: %s", tufUpdateRoot)
 	}
 
+	if runtime.GOOS == "linux" {
+		// Ensure only one instance of Fleet Desktop is running at a time
+		lockFile, err := getLockfile()
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not secure lock file")
+		}
+		defer func() {
+			if err := lockFile.Unlock(); err != nil {
```

Comment:
What happens if the Fleet desktop process is SIGKILLed?

Reply:
The lock is released

Reply:
OK, so under no circumstances a lock file will be left wandering that could prevent Fleet desktop from starting? Checking in case we are solving a problem and creating other problems :)

Which is why I'm more on the side of actually fixing the killing of the processes than adding a new way of keeping only one Fleet desktop running.

Reply:
https://www.gnu.org/software/libc/manual/html_node/File-Locks.html#:~:text=Likewise%2C%20locks%20are%20released%20when,(see%20Creating%20a%20Process). The lock is closed when the files are closed by the OS on exit. We defer closing the file properly to be good citizens but it's not actually necessary